lib/rules/media-feature-name-unit-allowed-list/README.md

```diff
@@ -0,0 +1,69 @@
+# media-feature-name-unit-allowed-list
+
+Specify a list of allowed name and unit pairs within media features.
+
+<!-- prettier-ignore -->
+```css
+@media (width : 50em) {}
+/**     ↑         ↑
+ * This media feature name and these units */
+```
+
+## Options
+
+`object`: `{ "name": ["array", "of", "units"]|"unit" }`
+
+If a property name is surrounded with `"/"` (e.g. `"/height/"`), it is interpreted as a regular expression. This allows, for example, easy targeting of shorthands: `/height/` will match `height`, `min-height`, `max-height`, etc.
+
+Given:
+
+```json
+{
+  "width": "em",
+  "/height/": ["em", "rem"]
+}
+```
+
+The following patterns are considered problems:
```

Comment:
Should I also demonstrate the level 4 media queries? Unsure if that's the convention we've been using for other rules.